tests/src/util/runner.ts

```diff
@@ -56,11 +56,32 @@ export class DroneRunner implements DropHandler {
   async certRefresh(
     certs: KeyCertPair,
     natsPort: number,
-    pebble: PebbleResult
+    pebble: PebbleResult,
+    eab = false,
```

Comment:
Instead of passing an `eab` boolean, let's just assume `eab` is used if `eabOptions` is not `undefined`.